tests/test_tags.py

```diff
@@ -313,29 +314,16 @@ def test_is_manylinux_compatible_glibc_support(
         self, version, compatible, monkeypatch
     ):
         monkeypatch.setitem(sys.modules, "_manylinux", None)
-        monkeypatch.setattr(
-            tags,
-            "_have_compatible_glibc",
-            lambda major, minor: (major, minor) <= (2, 5),
+        monkeypatch.setattr(tags, "_get_glibc_version", lambda: (2, 5))
+        assert (
+            bool(tags._is_manylinux_compatible("manylinux1", "any", version))
+            == compatible
         )
-        assert bool(tags._is_manylinux_compatible("manylinux1", version)) == compatible
-
-    @pytest.mark.parametrize(
-        "version_str,major,minor,expected",
-        [
-            ("2.4", 2, 4, True),
-            ("2.4", 2, 5, False),
-            ("2.4", 2, 3, True),
-            ("3.4", 2, 4, True),
-        ],
-    )
-    def test_check_glibc_version(self, version_str, major, minor, expected):
-        assert expected == tags._check_glibc_version(version_str, major, minor)
```

Comment:
What was this test meant to cover? The parsing of the string via the regex? `_check_libc_version` no longer exists. Parsing is tested below

Reply:
Yes it was.